operations/node_management/Rebuild_NCNs/Re-add_Storage_Node_to_Ceph.md

```diff
@@ -179,17 +175,17 @@ Use the following procedure to re-add a Ceph node to the Ceph cluster.
 
     Example output:
 
-    ```
+    ```bash
     NAME                             HOST      STATUS         REFRESHED  AGE  VERSION  IMAGE NAME                        IMAGE     D              CONTAINER ID
     rgw.site1.zone1.ncn-s001.kvskqt  ncn-s001  running (41m)  6m ago     41m  15.2.8   registry.local/ceph/ceph:v15.2.8      553b0cb212c          6e323878db46
     rgw.site1.zone1.ncn-s002.tisuez  ncn-s002  running (41m)  6m ago     41m  15.2.8   registry.local/ceph/ceph:v15.2.8      553b0cb212c          278830a273d3
     rgw.site1.zone1.ncn-s003.nnwuqy  ncn-s003  running (41m)  6m ago     41m  15.2.8   registry.local/ceph/ceph:v15.2.8           553b0cb212c      a9706e6d7a69
     ```
 
-1. Add nodes into HAproxy and KeepAlived.
+1. Add nodes into `HAproxy` and `KeepAlived`.
 
    ```bash
-   pdsh -w ncn-s00[1..(end node number)] -f 2 '/srv/cray/scripts/metal/generate_haproxy_cfg.sh; systemctl restart haproxy.service; /srv/cray/scripts/metal/generate_keepalived_conf.sh; systemctl restart keepalived.service'
+   pdsh -w ncn-s00[1-(end node number)] -f 2 'source /srv/cray/scripts/metal/update_apparmor.sh ; reconfigure-apparmor; /srv/cray/scripts/metal/generate_haproxy_cfg.sh > /etc/haproxy/haproxy.cfg; systemctl enable haproxy.service; systemctl restart haproxy.service; /srv/cray/scripts/metal/generate_keepalived_conf.sh > /etc/keepalived/keepalived.conf; systemctl enable keepalived.service; systemctl restart keepalived.service'
```

Comment:
Since the command is inside single quotes, we can use newlines here, making it more readable IMO. Since it's still a single copy-and-paste, the breaking it up into multiple lines like this doesn't make it harder on the user to perform.

```suggestion
   ncn-s# pdsh -w ncn-s00[1-(end node number)] -f 2 \
                   'source /srv/cray/scripts/metal/update_apparmor.sh
                    reconfigure-apparmor; /srv/cray/scripts/metal/generate_haproxy_cfg.sh > /etc/haproxy/haproxy.cfg
                    systemctl enable haproxy.service
                    systemctl restart haproxy.service
                    /srv/cray/scripts/metal/generate_keepalived_conf.sh > /etc/keepalived/keepalived.conf
                    systemctl enable keepalived.service
                    systemctl restart keepalived.service'
```